gammapy/spectrum/reflected.py

```diff
@@ -414,3 +419,122 @@ def plot(self, fig=None, ax=None, cmap=None, idx=None, add_legend=False):
             ax.legend(handles=handles)
 
         return fig, ax, cbar
+
+
+class ReflectedRegionsBackgroundMaker:
+    """Reflected regions background maker.
+
+    Parameters
+    ----------
+    region: `~regions.SkyRegion`
+        On region to compute off regions for.
+    angle_increment : `~astropy.coordinates.Angle`, optional
+        Rotation angle applied when a region falls in an excluded region.
+    min_distance : `~astropy.coordinates.Angle`, optional
+        Minimal distance between two consecutive reflected regions
+    min_distance_input : `~astropy.coordinates.Angle`, optional
+        Minimal distance from input region
+    max_region_number : int, optional
+        Maximum number of regions to use
+    exclusion_mask : `~gammapy.maps.WcsNDMap`, optional
+        Exclusion mask
+    binsz : `~astropy.coordinates.Angle`
+        Bin size of the reference map used for region finding.
+    """
+
+    def __init__(
+        self,
+        region,
+        angle_increment="0.1 rad",
+        min_distance="0 rad",
+        min_distance_input="0.1 rad",
+        max_region_number=10000,
+        exclusion_mask=None,
+        binsz="0.01 deg",
```

Comment:
If there is a default value, it is better to state it in the docstring no?

Reply:
I think at some point we decided (not sure if it's documented somewhere), that it's easier to maintain, if the default values are not repeated in the docstring again. Users always see the defaults together with the docstring when they use e.g. `help(ReflectedRegionsBackgroundMaker)` or `ReflectedRegionsBackgroundMaker?` in interactive environments. The same goes for the online documentation see e.g. [here](https://docs.gammapy.org/0.14/api/gammapy.spectrum.ReflectedRegionsBackgroundEstimator.html#gammapy.spectrum.ReflectedRegionsBackgroundEstimator). So the information would be basically repeated...